crates/autopilot/src/solvable_orders.rs

```diff
@@ -252,6 +252,12 @@ impl SolvableOrdersCache {
 
         let surplus_capturing_jit_order_owners = cow_amms
             .iter()
+            .filter(|cow_amm| {
```

Comment:
A metric for filtered out CoW orders (similar to filter reasons for user orders) would be helful imo